chaco/polar_mapper.py

```diff
@@ -40,27 +43,38 @@ class PolarMapper(AbstractMapper):
     def map_screen(self, data_array):
         """map_screen(data_array) -> screen_array
 
-        Converts radius and theta values from *data_array*
-        to x and y values and then maps
-        values from data space into screen space.
+        Overrides AbstractMapper. Maps values from data space into screen space.
         """
         self._compute_scale()
         if self._null_data_range:
-            return array([self.low_pos] * len(data_array))
+            if isinstance(data_array, (tuple, list, ndarray)):
+                return full_like(data_array, self.low_pos, dtype=float64)
+            else:
+                return array([self.low_pos])
```

Comment:
I'm not sure why these changes were made. Were these changes copied over from the linear mapper?

Reply:
I have the same comment about the changes in `map_data` and `_compute_scale`